src/migration/1710417425669-UserPersonalizedDigestSendType.ts

```diff
@@ -6,9 +6,17 @@ export class UserPersonalizedDigestSendType1710417425669 implements MigrationInt
     public async up(queryRunner: QueryRunner): Promise<void> {
         await queryRunner.query(`ALTER TABLE "user_personalized_digest" ADD "flags" jsonb NOT NULL DEFAULT '{}'`);
         await queryRunner.query(`CREATE INDEX "IDX_user_personalized_digest_flags_sendType" ON "user_personalized_digest" ((flags->>'sendType'))`);
+        await queryRunner.query(`
+            CREATE OR REPLACE FUNCTION clamp_to_hours(num NUMERIC) RETURNS NUMERIC
+                LANGUAGE SQL
+                IMMUTABLE
+                PARALLEL SAFE
+                RETURN ((num % 24) + 24) % 24;
+        `)
```

Comment:
@idoshamun for sanity check, did some more testing and I had to add a function to clamp offset to 0-23 since it would go outside of bounds depending on current UTC time and offset of timezone +/-. Actually caught it on tests. 

Set it to IMMUTABLE so result can be reused for multiple rows since we always send numeric offset.